subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -215,9 +213,8 @@ abstract class InHouseInstrumentationTestRunner :
     }
 
     override fun beforeOnCreate(arguments: Bundle) {
-        instrumentationArguments = arguments
         injectTestMetadata(arguments)
-        logger.debug("Instrumentation arguments: $instrumentationArguments")
+        logger.debug("Instrumentation arguments: $arguments")
```

Comment:
I moved this logging in parallel PR, is it ok?
Reason is to log closer to parsing implementation

https://github.com/avito-tech/avito-android/pull/893/files#diff-770e4a1f1e0f6067f45c21a4aa929158355f3b06e24be274c771af7a2592d396

Reply:
I will wait for your PR to fix conflicts